Scripts/script-Base64ListToFile.yml

```diff
@@ -13,12 +15,17 @@ script: |-
 
   # get the list
   res = demisto.executeCommand('getList', {'listName': demisto.args()['listname']})
+  if res['Type'] == entryTypes['error']:
+    raise Exception("error reading list %s from demisto" % demisto.args()['listname'])
```

Comment:
Please use `return_error` function and remove `sys.exit(0)`

Reply:
V